doc/user/content/sql/explain-plan.md

```diff
@@ -415,7 +420,7 @@ EXPLAIN WITH(types) FOR
 SELECT a.id, sum(b.amount) FROM accounts a JOIN bids b ON(a.id = b.buyer) GROUP BY a.id;
 ```
 
-Explain the physical plan as text:
+Explain the physical plan as verbose text:
```

Comment:
Because we mention "verbose" here .... could we disambiguate in the preceding snippets of "Same as above, ... verbose"?

Also, just seeing that line 405 is missing FOR 😄